polling_stations/templates/fragments/errors.html

```diff
@@ -3,7 +3,7 @@
 
 <div class="ds-text-centered">
     <h2>{% blocktrans %}Sorry, we can't find {{ postcode }}{% endblocktrans %}</h2>
-    <p>{% trans "This doesn't appear to be a valid postcode." %}</p>
+    <p>{% trans "We don't hold data for this postcode, or it is an invalid postcode." %}</p>
```

Comment:
I haven't remade the translation files, because when I ran `makemessages` there were loads of things that had changed. I think it's better that we just deal with it all together after May 1st.

Reply:
Yep. Agreed.
I was planning on getting https://app.asana.com/1/1204880536137786/project/1205090401342417/task/1208714863410843?focus=true done before this election cycle tbh, but it never fitted in.

Reply:
Can we say "we don't hold information..." here rather than "data". It feels a bit less techy